### DISCUSSION TITLE
مساعدة

### DISCUSSION THREAD

**Discussion by Samirdahmani** [community/community#41496] [New to GitHub]
مرحبا من فظلكم كيف يعمل هذا البرنامج

**Comment by DSDmark** [comment]
Hi @Samirdahmani,
   
> Nice to meet you and welcome to this great community, we are here to help each other and try to make open source better. This is the discussions section if you want to discuss any. you can ask here, we will try to help.

And also, why any chance you are curious about, 

> what is GitHub, who is GitHub then I highly suggest you check this out [**What is GitHub?**](https://youtu.be/pBy1zgt0XPc). It's a video. so, just ***sit back and relax***

About the **GitHub** in short way,

> it's a place for anyone from anywhere, to build anything. **anything** you think the world needs. 

![image](https://user-images.githubusercontent.com/85744816/206890536-5f5d663a-9284-4a0a-b189-8f244254bbac.png)

**Comment by mohamedzhioua** [comment]
@Samirdahmani هذه دورة كاملة عن كيفية استخدام github في اللغة العربية
https://www.youtube.com/watch?v=ACOiGZoqC8w&list=PLDoPjvoNmBAw4eOj58MZPakHjaO3frVMF&ab_channel=ElzeroWebSchool
